### PR TITLE
UPDATE : RTL-ing main elements

### PR DESCRIPTION
**Pull request by JTS-IL**
A very long walk-trough /rtl/tc_common.css with small adjustments in some php code files that now support both RTL and LTR sites without too much css tricks

### CODE REVIEW

**Comment by JTS-IL** [comment]
BTW, I'm quite new to git so I hope every thing is staged/committed/pulled the right way...

**Comment by JTS-IL** [comment]
BTW, I'm quite new beginner with git, so I hope I've done everything the *right* way :-)
All the changes are implemented on prod site: www.glezer.co.il

File: inc/parts/class-content-sidebar.php [hunk 1]
```diff
@@ -35,14 +35,15 @@ function __construct () {
...
+		add_action ( '__before_left_sidebar'       , array( $this , 'tc_social_in_sidebar' ) );
+ 
         //displays right sidebar
-        add_action ( '__after_article_container'   , array( $this , 'tc_sidebar_display' ) );
-        add_action ( '__before_right_sidebar'      , array( $this , 'tc_social_in_sidebar' ) );
+		add_action ( '__after_article_container'   , array( $this , 'tc_sidebar_display' ) );
+		add_action ( '__before_right_sidebar'      , array( $this , 'tc_social_in_sidebar' ) );
+
 
-        //since 3.2.0 show/hode the WP built-in widget icons
```
Comment (Nikeo) [inline inc/parts/class-content-sidebar.php]: Thanks for fixing this typo :)

Comment (JTS-IL) [inline inc/parts/class-content-sidebar.php]: ;-)

**Comment by Nikeo** [comment]
Excellent Yaacov !
Your code has been merged.
I've adapted the footer part to 'normalize' it a little bit for handling the conditional priority depending on rtl.
Thanks :)

**Comment by JTS-IL** [comment]
Happy to help the open source community :-)

**Comment by Nikeo** [comment]
I've finish the RTL update and merged it in the dev branch.

For future modifications, you'll want to modify the less files in inc/assets/less which are used to generate the different stylesheets, including the rtl one.

As you can see in the code, some less variables are rtl related like @is_rtl.

More about less : http://lesscss.org/

Thanks again for this contribution :+1:

**Comment by JTS-IL** [comment]
Well, i"m not familiar with LESS, so I will start learning it soon. :-)

**Pull request closed without merging**